README.md

```diff
@@ -17,13 +17,15 @@ Notes:
 
 ```
 module "lambda_ami_backup" {
-  source = "git::https://github.com/cloudposse/tf_lambda_ami_backup.git?ref=master"
+  source = "git::https://github.com/cloudposse/tf_ami_backup.git?ref=master"
```

Comment:
Pin to the tag which will be added after this PR merged.
I think, we should do major tag version change, because logic of the module was heavily modified.